python/letsql/examples/core.py

```diff
@@ -0,0 +1,36 @@
+import functools
+import pathlib
+
+import letsql as ls
+
+
+whitelist = [
+    "astronauts",
+    "awards_players",
+    "batting",
+    "diamonds",
+    "functional_alltypes",
+    "iris",
+    "penguins",
+]
+
+
+@functools.cache
+def get_name_to_suffix():
+    board = ls.options.pins.get_board()
+    dct = {
+        name: pathlib.Path(board.pin_meta(name).file).suffix
+        for name in board.pin_list()
+        if name in whitelist
+    }
+    return dct
+
+
+def get_table_from_parquet(name, backend, table_name):
```

Comment:
can we have `table_name` default to `None` and use the value of `name` if not given?